modules/aws_account_permissions/main.tf

```
@@ -0,0 +1,208 @@
+terraform {
+  required_providers {
+    illumio-cloudsecure = {
+      source  = "illumio/illumio-cloudsecure"
+      version = "~> 1.0.10"
```

Update to the latest release.
```suggestion
      version = "~> 1.0.11"
```
Update it also in the other existing submodule:
https://github.com/illumio/terraform-illumio-cloudsecure/blob/9955a42394910d48a1bed1af7ea5dc201200db61/modules/k8s_cluster/versions.tf#L5